src/test/storage.zig

```diff
@@ -116,14 +116,14 @@ pub const Storage = struct {
         assert(options.write_latency_mean >= options.write_latency_min);
```

Comment:
To be explicit around the size of data file that we support, regardless of architecture.

In other words, `usize` is more for where we explicitly want to use a machine word-sized integer, for example, for performance when writing an encoding/decoding routine that processes data in the data plane using word-sized integers—think of all the tricks that Daniel Lemire uses around machine words.

Whereas, here, for `offsets/sizes`, we're interested in addressing data files larger than 4GiB. i.e. We don't mind using two machine words on a 32-bit architecture.

Where it's not in the data plane, and where there's no specific performance reason to do otherwise, `u64` is generally almost always a better bet than `usize` since it's not going to overflow, and since it's not subject to non-determinism based on architecture.

Where memory is a concern, and where `u32` is the right choice, then that's also better. We save memory on 64-bit architectures, and we're also still more explicit and deterministic than a `usize`.

I know that Zig's std lib almost always just uses `usize` but I prefer that this is reserved only for where the intention is literally to use machine words in the data plane. For the control plane, `u64` (default) or `u32` (memory-intensive) are better deterministic choices. They're also more explicit.